Assets/Scripts/FootstepsSounds.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FootstepsSounds : MonoBehaviour
+{
+    public CharacterController playerController;
+    public AudioSource audioSource;
+    // private float walkStepRate = 0.08f;
+    // private float runStepRate = 0.02f;
+    // AudioSource
+    // Start is called before the first frame update
+    void Start()
+    {   
+        //Assign audiosource
+        audioSource = gameObject.GetComponent<AudioSource> ();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {   
+        //If player is grounded and moving then play sound
+        //High pitched sound for player running fast
+        //Random ranges of pitch and volume to add variation to step sounds
+        if (playerController.isGrounded && Input.GetKey("left shift") && !audioSource.isPlaying) {
```

Comment:
This is great we should also use this for player movement logic

Reply:
Happy to implement the movement tomorrow. 

Just tried to fix conflicts but just got really complicated so pushed from a new branch (footsteps3) but only the script and the audio- can update the prefab when we integrate.